app/sidecar/rev-c-lab.toml

```diff
@@ -3,3 +3,4 @@ inherit = "rev-c.toml"
 [patches]
 name = "sidecar-c-lab"
 features.sequencer = ["stay-in-a2"]
+features.ignition = ["always-transmit"]
```

Comment:
I think we would also want this in the `rev-b-lab.toml`, right?

Reply:
Yes, this should be present in both lab images.